usagov/usagov-2021
# usagov-1953-incorrect-status-announced: update code to add a class at…

… form submission


## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1953

## Description

I updated the reportAProblem javascript file to add a class to the form upon submission so the error doesn't appear if there are no errors
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


Goto http://localhost/site-issue-report-form and http://localhost/es/reporte-problemas-en-este-sitio-web. 
1. On each page do not fill out the form to make sure the error box appears at the top of the form of the with the number of errors
2. Fill out on error field and make sure the number of errors in the error box changes to "2"
3. Fill out another field so now 2 fields will be filled out and make sure the number in the error box changes "1"
4. Fill out all fields and submit the form and make sure the error box doesn't appear at the top of the form.

Screenshot of current form with error box after submission with no errors
<img width="1512" alt="Screenshot 2024-10-18 at 11 31 47 AM" src="https://github.com/user-attachments/assets/dfbcfab7-f630-4b4e-9bac-35e5e5cefe47">

Screenshot of updated form with no error box after submission no errors
<img width="1512" alt="Screenshot 2024-10-18 at 11 42 05 AM" src="https://github.com/user-attachments/assets/22792a91-0b96-42f9-8b20-be2866cb30b7">

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
6. Find the commit of this pull request.
7. Build and deploy the changes.
8. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
